CHANGELOG.md

```diff
@@ -0,0 +1,85 @@
+# Changelog
+
+## [Unreleased]
+<details>
+  <summary>
+    Changes that have landed in master but are not yet released.
+    Click to see more.
+  </summary>
+
+### Components
+
+- `bblfsh/bblfshd` has been updated to [v2.12.0-drivers](https://github.com/bblfsh/bblfshd/releases/tag/v2.12.0).
+
+### New Features
+
+- `srcd prune` now removes the gitbase index data ([#352](https://github.com/src-d/engine/issues/352)).
+- More friendlier and useful error messages ([#252](https://github.com/src-d/engine/issues/252), [#258](https://github.com/src-d/engine/issues/258), [#272](https://github.com/src-d/engine/issues/272), [#291](https://github.com/src-d/engine/issues/291), [#294](https://github.com/src-d/engine/issues/294), [#295](https://github.com/src-d/engine/issues/295), [#326](https://github.com/src-d/engine/issues/326)).
+- Replace the basic REPL sql shell with a standard mysql client ([#154](https://github.com/src-d/engine/issues/154)).
+- Show the container exposed ports in the `srcd components list` output ([#300](https://github.com/src-d/engine/issues/300)).
+
+### Bug Fixes
+
+- Fix the gRCP `ResourceExhausted` error for `srcd parse uast` with big files ([#271](https://github.com/src-d/engine/issues/271)).
+- In cases where the workdir contains a lot of Git repositories, `srcd web sql` could open the web UI before gitbase was ready to accept queries ([#284](https://github.com/src-d/engine/issues/284)).
```

Comment:
also:
```suggestion
- In cases where the workdir contains a lot of Git repositories, `srcd web sql` could open the web UI before gitbase was ready to accept queries ([#284](https://github.com/src-d/engine/issues/284)).
- Set ConsistencyDelegated mode for volumes on macOs always to improve OSX gitbase performance. Caveat: in case of container crash, indexes can be written incorrectly, but gitbase will drop them on next init ([#330](https://github.com/src-d/engine/issues/330))
```